src/components/CopyableInput/CopyableInput.js

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+	Button, Input, InputGroup
+} from 'reactstrap';
+
+import './CopyableInput.scss';
+
+
+/*
```

Comment:
Would be nice to see the usage here as well (or in README)

Reply:
example added